setup.py

```diff
@@ -2,9 +2,19 @@
 
 from os.path import exists
 from setuptools import setup
+import re
+
+
+version_raw = open('chest/_version.py').read()
+version_regex = r"^__version__ = ['\"]([^'\"]*)['\"]"
+version_result = re.search(version_regex, version_raw, re.M)
+if version_result:
+    version_string = version_result.group(1)
+else:
+    raise RuntimeError("Unable to find version string in %s." % (VERSIONFILE,))
```

Comment:
`os.path.join('..', 'version.txt')` doesn't work because `.` is the current working directory, which depends on where its being invoked from.  I think this helps explain why the SO solution make the version file a python file: the interpreter knows where `__init__.py` is located and locates `._version` based on that.